src/AWS/Orleans.Streaming.SQS/Hosting/ClientBuilderExtensions.cs

```diff
@@ -14,32 +14,36 @@ public static class ClientBuilderExtensions
         /// </summary>
         public static IClientBuilder AddSqsStreams(this IClientBuilder builder, string name, Action<SqsStreamOptions> configureOptions)
         {
-            return builder.ConfigureServices(services => services.AddClusterClientSqsStreams(name, configureOptions));
+            return builder
+                .ConfigureApplicationParts(parts => parts.AddFrameworkPart(typeof(SQSAdapterFactory).Assembly))
```

Comment:
Due to the previous pattern we've many overrides for adding things to the service collection that are not necessary if we're only supporting the builders (which I agree), but the advantage of the previous pattern was that everything eventually ended in one place with a single authoritative set of logic that did the work.

In changing these, I advocate that the options action override call through to the options action builder override, which has the single authoritative set of logic to run.  It should add the parts, and setup the services.

Reply:
Sure thing, fixed in latest